static/styles/settings.css

```diff
@@ -569,9 +573,28 @@ input[type="checkbox"] {
     margin-top: 10px;
 }
 
-.language_selection_widget .language_selection_button {
-    text-decoration: none;
-    margin-left: 20px;
+.language_selection_widget {
+    margin-bottom: 20px;
+
+    .title {
+        margin-bottom: 5px;
+    }
+
+    .language_selection_button {
+        text-decoration: none;
+        color: hsl(0, 0%, 20%);
+    }
+}
+
+.realm_default_language {
+    .input-group {
+        display: inline;
+    }
+
+    .help_link_widget_fake_label {
+        display: inline-block;
```

Comment:
Not sure if we actually use this CSS. Cannot find such element.

Reply:
Yup.